docs_src/security/Ch-CORS-Settings.md

```diff
@@ -0,0 +1,29 @@
+# CORS enabling
+
+!!! edgey "EdgeX 2.1"
+    New for EdgeX 2.1 is the ability to enable CORS access to EdgeX microservices through configuration. 
+
+The EdgeX microservices provide REST APIs and those services might be called from a GUI through a browser. Most browsers prevent service calls from a different origin. Thus, EdgeX supports Cross-Origin Resource Sharing (CORS) since Jakarta release (v2.1), and this feature can be controlled by the configurations. The default behavior of CORS is disabled. Here is a good reference to understand [CORS](https://www.html5rocks.com/en/tutorials/cors//).
+
+!!! note
+    C Device SDK doesn't support CORS, and enabling CORS in Device Services is not recommended because browsers should not access Device Services directly.
+
+
+## CORS configurations
+
+The CORS settings are under `Service.CORSConfiguration` configuration, and they can be set via `Service_CORSConfiguration_*` environment variables.
+
+=== "Service.CORSConfiguration"
+|Property|Default Value|Description|
+|---|---|---|
+|||The settings of controling CORS http headers|
+|EnableCORS|false|Enable or disable CORS support.|
+|CORSAllowCredentials|false|The value of `Access-Control-Allow-Credentials` http header. It appears only if the value is `true`.|
+|CORSAllowedOrigin|"https://localhost"|The value of `Access-Control-Allow-Origin` http header.|
+|CORSAllowedMethods|"GET, POST, PUT, PATCH, DELETE"|The value of `Access-Control-Allow-Methods` http header.|
+|CORSAllowedHeaders|"Authorization, Accept, Accept-Language, Content-Language, Content-Type, X-Correlation-ID"|The value of `Access-Control-Allow-Headers` http header.|
+|CORSExposeHeaders|"Cache-Control, Content-Language, Content-Length, Content-Type, Expires, Last-Modified, Pragma, X-Correlation-ID"|The value of `Access-Control-Expose-Headers` http header.|
+|CORSMaxAge|3600|The value of `Access-Control-Max-Age` http header.|
```

Comment:
This section should be add to the common configuration doc and referenced from here.

Reply:
moved this section to common configuration page